src/middlewares.js

```diff
@@ -319,6 +319,9 @@ export function handleParseErrors(err, req, res, next) {
       case Parse.Error.OBJECT_NOT_FOUND:
         httpStatus = 404;
         break;
+      case Parse.Error.INVALID_SESSION_TOKEN:
```

Comment:
humm.. I'm not sure... I think it can break some apps. Let's just leave 400 as it was before.